views/resources/index.ejs

```diff
@@ -1,48 +1,31 @@
 <%- contentFor('title') %>
 <%- contentFor('body') %>
-<head>
 <u><h1>Resources Page</h1></u>
-<style>
-  * {
-    box-sizing: border-box;
-  }
-  .column {
-    float: left;
-    width: 50%;
-    padding: 10px;
-    height: 300px;
-  }
-  .row:after {
-    content: "";
-    display: table;
-    clear: both;
-  }
-  </style>
-  </head>
-  <body>
+<body style="background-color:white;">
 
 <ul>
   <%for(let record of records){%>
 
-    <li>
-      <div class="row">
-        <div class="column">
-         <% if (record.logo) { %>
-
-
-      <img src="<%= assetUrl(record.logo) %>">
-      <% } %>
-      </div>
-        <div class="column">
-          <%=record.name %>
-          <%=record.orgtype%>
-          <%=record.contactperson %>
-          <%=record.phone%>
-          <%=record.address%>
-          <%=record.email%>
-          <%=record.website%></li>
-          </div>
-        </div>
+    <div class="container">
```

Comment:
You shouldn't need the div container INSIDE the for loop... instead, put it outside of the ul. Or, put the "container" class on the ul tag...